### ISSUE TITLE
Proper Contrast Ratios 

### ISSUE THREAD

**Issue by wkaisertexas** [thecourseforum/theCourseForum2#681]
Proper contrast ratios are important. The current stylesheet is way to hard to read and does not meet web accessibility requirements.

# Proposed 
<img width="1089" alt="Screenshot 2023-09-25 at 12 48 35 AM" src="https://github.com/thecourseforum/theCourseForum2/assets/27795014/8b2ec44c-8a7b-4a2b-b55d-bab3dd6b0b56">

4.5:1 constrast ratio between text and background makes content easier to read [as per the W3C](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#:~:text=The%20visual%20presentation%20of%20text,Incidental)

# Currently
<img width="1096" alt="Screenshot 2023-09-25 at 12 48 56 AM" src="https://github.com/thecourseforum/theCourseForum2/assets/27795014/eddcba30-418e-43cc-97dd-d5d36532ba63">

Contrast between text and background is insufficient to allow for reading without significant eye strain. 

---

The colors shown are not final and coordination with the UX / Design team will be required before any changes can be made. 

Additionally, because we are a CIO, [UVA Brand Colors](https://brand.virginia.edu) are something we could use (official uva blue, official uva orange, etc.)
